### ISSUE TITLE
Documentation: Add installation instruction

### ISSUE THREAD

**Issue by homersimpsons** [exercism/configlet#752]
I cannot seem to find any installation instruction.

None of those contains it:
- `README.md` https://github.com/exercism/configlet#readme
- Exercism's doc https://exercism.org/docs/building/configlet
- Search for any INSTALLATION.md, CONTRIBUTING.md file in this repo

It looks like the process is:
1. Go to relases: https://github.com/exercism/configlet/releases
2. Download the release for your OS
3. Extract the dowloaded file
4. Use the binary in the release

## Answer
Forum link: https://forum.exercism.org/t/configlet-documentation-add-installation-instruction/5363/7

TL;DR:
- Use `bin/fetch-configlet` or download it for releases
- In README it is under the "Use it in your track" section (at the end)
- In Exercim's documentation it is in the "Installation" section (at the end)
